codalab/apps/api/views/worksheet_views.py

```diff
@@ -435,27 +435,40 @@ def post(self, request):
 
 class BundleUploadApi(views.APIView):
```

Comment:
What is the file size limit? We get questions on this all the time for competition bundles, submissions, etc. -- do we know for sure what it is here?